rest/route/github.go

```diff
@@ -667,6 +678,30 @@ func (gh *githubHookApi) requireSigned(ctx context.Context, userRepo data.UserRe
 	return nil
 }
 
+func (gh *githubHookApi) checkPRApprovals(ctx context.Context, settings *evergreen.Settings, userRepo data.UserRepoInfo, prNum, requiredApprovalCount int) error {
+	githubToken, err := settings.GetGithubOauthToken()
+	if err != nil {
+		return errors.Wrap(err, "getting GitHub OAuth token from settings")
+	}
+
+	reviews, err := thirdparty.GetGithubPullRequestReviews(ctx, githubToken, userRepo.Owner, userRepo.Repo, prNum)
+	if err != nil {
+		return errors.Wrap(err, "getting GitHub PR reviews")
+	}
+
+	var numApprovals int
+	for _, r := range reviews {
+		if r.GetState() == reviewApproved {
+			numApprovals += 1
+		}
+	}
+
+	if numApprovals < requiredApprovalCount {
+		return errors.Errorf("PR %d does not have enough approvals. '%s' approval(s) required", prNum, requiredApprovalCount)
```

Comment:
"%d approval(s) required" since `requiredApprovalCount` is an int.

Reply:
done